shared/src/lib.rs

```diff
@@ -12,6 +12,15 @@ pub struct DeployApiKeyInfo {
     pub timestamp: u64,
 }
 
+/// Request payload for the `/update-secret` and `/delete-secret` HTTP route
```

Comment:
```suggestion
/// Request payload for the `/delete-secret` HTTP route
```
There isn't an `/update-secret`, right?  If you want to update an api key you can just hit `deploy-api-key` a second time.